### PR TITLE
[16.0][MIG] purchase_supplier_rank (from 13.0)

### PR DESCRIPTION
**Pull request by amarcosg**

### CODE REVIEW

**Review by aliciagaarzo — APPROVED** [review]
Code review LGTM.

**Review by hugo-cordoba — APPROVED** [review]
LGTM! (Code Review)

**Comment by hugo-cordoba** [comment]
Hi @rousseldenis, could you take a look and merge? 

Thankss! Best regards!

File: purchase_supplier_rank/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module updates supplier_rank for partners when creating purchase orders.
```
Comment (NL66278) [inline purchase_supplier_rank/readme/DESCRIPTION.rst]: Maybe add a comment here that normally supplier_rank is only updated by creating vendor invoices.

Comment (rafaelbn) [reply purchase_supplier_rank/readme/DESCRIPTION.rst]: Please @amarcosg check this! Thank you!

Comment (amarcosg) [reply purchase_supplier_rank/readme/DESCRIPTION.rst]: Description updated, thanks for the review!

**Comment by kikopeiro** [comment]
Hi!

What about this PR? @rousseldenis could you merge?

Thankssssssss!

**Comment by amarcosg** [comment]
Hi @rafaelbn!

Could you please merge this PR? I have already resolved all the reviews.

Thanks!

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**